explorer/app/components/SocialLinks/SocialLinks.tsx

```diff
@@ -1,7 +1,7 @@
 import Link from "next/link";
 import React from "react";
 
-export type SocialType = "twitter" | "github" | "youtube" | "slack";
+export type SocialType = "twitter" | "github" | "youtube" | "slack" | "help";
```

Comment:
```suggestion
export type SocialType = "twitter" | "github" | "youtube" | "slack" | "discourse";
```

Reply:
The "help" link for AnVIL is actually Discourse (although it's actually difficult to tell!). Would you be able to make the associated changes to `SocialLinkItem` etc?

Reply:
Sure!